perceval/backends/core/confluence.py

```diff
@@ -129,7 +130,7 @@ def fetch_items(self, category, **kwargs):
 
             for hc in hcs:
                 hc['content_url'] = content_url
-                hc['ancestors'] = Confluence.ancestors(hc, content)
+                hc['ancestors'] = self.__fetch_ancestors(content)
```

Comment:
according to the comment below, this line could be transformed into:
`hc['ancestors'] = content['ancestors'] if 'ancestors' in content else []`

Reply:
as minor comment, I would add a space after this statement to improve the readability (feel free to address this comment if you want)